app/views/coach_dashboard/show.html.erb

```diff
@@ -0,0 +1,14 @@
+<h1>Hello <%= current_user.type %> <%= current_user.first_name %></h1>
+
+<p class="institution">
+  <h2>Institution: <%= current_user.profile.institution %></h1>
```

Comment:
checkout your h2 tag... its closed with /h1. Also, no need to wrap h2's in <p> tags. What about a <div>?